website/static/js/pages/shareEmbedConfig.js

```diff
@@ -0,0 +1,7 @@
+var pym = require('pym.js');
```

Comment:
Where does `pym.js` come from? Looks like it needs to be added to `package.json`.